docs/modules/ROOT/pages/how-tos/aks_recover_from_backup.adoc

```diff
@@ -0,0 +1,141 @@
+= AKS Recover cluster objects from Azure Blob Storage backup
+
+== Prerequisites
+
+* Executables used in this guide:
+** `kubectl`
+** `yq`
```

Comment:
I'm guessing you are using v4 as well?
Some distros ship v3 or older, v4 is completely different though. I think we should really clarify this as all the commands could fail with a mismatching yq version.
```suggestion
** `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
```